tox.ini

```diff
@@ -0,0 +1,15 @@
+[tox]
+envlist = py27
+
+[base]
+deps =
+	coverage
+
+[testenv:py27]
+deps=
+	{[base]deps}
+	-r{toxinidir}/tests/requirements.txt
+
+commands =
+	coverage run --source mozdownload,test {toxinidir}/tests/test.py
```

Comment:
We don't need the coverage for `test`. It's only `mozdownload`. Also you should be able to get rid of `{toxinidir}` because tox always assumes that as root. So `test/test.py` should be enough.